bigbluebutton-client/src/org/bigbluebutton/modules/present/ui/views/PresentationWindow.mxml

```diff
@@ -430,21 +430,8 @@ with BigBlueButton; if not, see <http://www.gnu.org/licenses/>.
 			}
 										
 			private function disableSlideNavigationButtons(pageNumber:int):void {
-				if (pageNumber == 1) {
-					backButton.mouseEnabled = false;
-					backButton.styleName = "presentationBackButtonDisabledStyle";
-				} else {
-					backButton.mouseEnabled = true;
-					backButton.styleName = "presentationBackButtonStyle";
-				}
-				
-				if (pageNumber < PresentationModel.getInstance().getNumberOfPages()) {
-					forwardButton.mouseEnabled = true;
-					forwardButton.styleName = "presentationForwardButtonStyle";
-				} else {
-					forwardButton.mouseEnabled = false;
-					forwardButton.styleName = "presentationForwardButtonDisabledStyle";
-				}
+				backButton.enabled = backButton.mouseEnabled = pageNumber != 1; 
+				forwardButton.enabled = forwardButton.mouseEnabled = pageNumber < PresentationModel.getInstance().getNumberOfPages();

```

Comment:
It needs to be mouseEnabled. If you use enabled the tab position is reset when the button is disabled.